app/logic/spreadsheet.py

```diff
@@ -7,16 +7,14 @@
 from app import app
 from app.models.eventParticipant import EventParticipant
 from app.models.user import User
-from app.models.programEvent import ProgramEvent
 from app.models.program import Program
 from app.models.event import Event
 from app.models.term import Term
 
 
 def volunteerHoursByProgram():
     query = ((Program.select(Program.programName, fn.SUM(EventParticipant.hoursEarned).alias('sum'))
-                     .join(ProgramEvent)
-                     .join(EventParticipant, on=(ProgramEvent.event == EventParticipant.event))
+                     .join(EventParticipant, on=(Event.program== EventParticipant.event))
                      .group_by(Program.programName)
                      .order_by(Program.programName)))
```

Comment:
This query doesn't work anymore. You can't use `Event.program` in the join condition because Event is not included in the query